config/environments/test.rb

```diff
@@ -14,6 +14,11 @@
 
   config.assets.digest = false
 
+  config.public_file_server.enabled = true
+  config.public_file_server.headers = {
+    'Cache-Control' => "public, max-age=#{1.hour.to_i}",
+  }
```

Comment:
What is it used for?

Reply:
Right, though we use our own middleware for this: https://github.com/mastodon/mastodon/blob/9e245d147bcb2c72cc552ff8c276a1c34e2f686d/config/application.rb#L186-L189

I notice that it's not enabled by default in the test environment though.

Reply:
Good catch, I completely missed this!

Do you see any issue with enabling the middleware in tests as well? Or should I try to find a way to have the environment variable set when Puma is launched for the system tests?

Reply:
I can't see any issue in enabling it for tests as well.